src/Bridge/Doctrine/Orm/Filter/AbstractFilter.php

```diff
@@ -173,4 +174,29 @@ protected function extractProperties(Request $request) : array
 
         return $request->query->all();
     }
+
+    /**
+     *
+     * @param $property
+     * @param $alias
+     * @param $queryBuilder
+     *
+     * @return array
+     */
+    protected function getAssociation($property,$alias,&$queryBuilder) : array
```

Comment:
It's not necessary to specify pass by reference for `$queryBuilder`. Objects are always passed by reference.